cfgov/v1/models.py

```diff
@@ -122,12 +242,12 @@ class BlogPage(V1Page):
         related_name='+'
     )
 
-    content_panels = Page.content_panels + [
+    content_panels = V1Page.content_panels + [
         FieldPanel('date'),
         FieldPanel('body', classname="full"),
     ]
 
     promote_panels = [
-        MultiFieldPanel(Page.promote_panels, "Common page configuration"),
+        MultiFieldPanel(V1Page.promote_panels, "Common page configuration"),
```

Comment:
@anselmbradford How about the name CFGOVPage? Or CFPage? Or CFPBPage? Or FlapjackPage? RefreshPage?